ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/TopKDistinctTestUtils.java

```diff
@@ -16,7 +16,9 @@
 package io.confluent.ksql.function.udaf.topkdistinct;
 
 import io.confluent.ksql.function.AggregateFunctionInitArguments;
+import io.confluent.ksql.schema.ksql.SqlArgument;
 import io.confluent.ksql.schema.ksql.types.SqlType;
+
```

Comment:
Does the IDE add these random extra lines?

Reply:
Not sure where this came from, I removed it though